persistence/src/main/java/com/epam/rft/atsy/persistence/entities/ChannelEntity.java

```diff
@@ -22,18 +22,14 @@
 @ToString(callSuper = true)
 @Entity
 @Table(name = "channels", schema = "atsy", uniqueConstraints = @UniqueConstraint(columnNames = "name"))
-public class ChannelEntity extends SuperEntity implements java.io.Serializable {
+public class ChannelEntity extends LogicallyDeletableEntity implements java.io.Serializable {
 
-
-  @Column(name = "name", nullable = false, length = 255, unique = true)
```

Comment:
No, but 255 is the default value, so I think it was not necessary to define again.